src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/Win32/UnsafeNativeMethodsTablet.cs

```diff
@@ -120,33 +119,33 @@ public override bool IsInvalid
                 return IsClosed || handle == IntPtr.Zero;
             }
         }
-  
-        override protected bool ReleaseHandle()
+
+        protected override bool ReleaseHandle()
         {
-            //Note: It is not an error to have already called DestroyRecognizer
-            //which makes _recognizerHandle.IsInvalid == true before calling
-            //DestroyContext.  I have removed this assert, but left it commented for
-            //context.
-            //Debug.Assert(_recognizerHandle != null && !_recognizerHandle.IsInvalid);
-            Debug.Assert(handle != IntPtr.Zero);
-            int hr = MS.Win32.Recognizer.UnsafeNativeMethods.DestroyContext(handle);
+            // Note: It is not an error to have already called DestroyRecognizer
+            // which makes _recognizerHandle.IsInvalid == true before calling
+            // DestroyContext.  I have removed this assert, but left it commented for
+            // context.
+            //
+            // Debug.Assert(_recognizerHandle != null && !_recognizerHandle.IsInvalid);
+            Debug.Assert(handle != 0);
+            HRESULT result = UnsafeNativeMethods.DestroyContext(handle);
 
             // Now, dereference the attached recognizer.
             _recognizerHandle = null;
 
-            return MS.Internal.HRESULT.Succeeded(hr);
```

Comment:
Double checking - how did this define success? Was S_FALSE included?

Reply:
>= 0